main-labs/sdn-openflow/pox/01-pox-controller/README.md

```diff
@@ -32,11 +32,13 @@ ovs-ofctl dump-flows s1
 
 This happens since the switch acts as a L2 learning switch by the default, but does not fill the table.
 
-To enable the `forwarding.l2_learning` POX component and see the flow rules in the switch, go into the `controller`
-terminal and type:
+To enable the `forwarding.l2_learning` POX component and see the flow rules in the switch, launching ```kathara connect controller```.
+
+Launch in the root@controller:
 ```
-python3.9 /pox/pox.py forwarding.l2_learning openflow.of_01 -port=6653
+python3 /pox/pox.py forwarding.l2_learning openflow.of_01 -port=6653
```

Comment:
Maybe is better to leave here `python3.9`, since it is the version supported by POX